### ISSUE TITLE
Error with get note(s)

### ISSUE THREAD

**Issue by Soleone** [jonhadfield/sn-cli#4]
Getting the following error when trying to get all notes or a specific note by title:

```
➜ sn get notes
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x143af15]

goroutine 1 [running]:
main.startCLI.func18(0xc0000f7080, 0x0, 0xc00011a120)
	/home/circleci/project/cmd/sncli/main.go:885 +0xd75
github.com/urfave/cli.HandleAction(0x1496760, 0xc0000911a0, 0xc0000f7080, 0xc0000f7080, 0x0)
	/go/pkg/mod/github.com/urfave/cli@v1.22.1/app.go:523 +0xbe
github.com/urfave/cli.Command.Run(0x152d612, 0x4, 0x0, 0x0, 0xc00012cbe0, 0x1, 0x1, 0x15315f0, 0x9, 0x0, ...)
	/go/pkg/mod/github.com/urfave/cli@v1.22.1/command.go:174 +0x509
github.com/urfave/cli.(*App).RunAsSubcommand(0xc000122380, 0xc0000f6dc0, 0x0, 0x0)
	/go/pkg/mod/github.com/urfave/cli@v1.22.1/app.go:404 +0x830
github.com/urfave/cli.Command.startApp(0x152cf3a, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0, 0x15315e7, 0x9, 0x0, ...)
	/go/pkg/mod/github.com/urfave/cli@v1.22.1/command.go:329 +0x839
github.com/urfave/cli.Command.Run(0x152cf3a, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0, 0x15315e7, 0x9, 0x0, ...)
	/go/pkg/mod/github.com/urfave/cli@v1.22.1/command.go:102 +0x9a1
github.com/urfave/cli.(*App).Run(0xc0001221c0, 0xc0000ac120, 0x6, 0x6, 0x0, 0x0)
	/go/pkg/mod/github.com/urfave/cli@v1.22.1/app.go:276 +0x6c4
main.startCLI(0xc0000ac120, 0x6, 0x6, 0xa2, 0xc000084180, 0x5e, 0xc000166000, 0xc000157f88)
	/home/circleci/project/cmd/sncli/main.go:1336 +0x33e4
main.main()
	/home/circleci/project/cmd/sncli/main.go:42 +0x57
```

**Comment by jonhadfield** [comment]
It's trying to output notes that have been deleted (flagged as deleted and content removed for the item). Will get a fix out asap.

**Comment by jonhadfield** [comment]
Fixed in release 0.0.15:
https://github.com/jonhadfield/sn-cli/releases/tag/0.0.15.
